examples/07_ken_embeddings_example.py

```diff
@@ -0,0 +1,304 @@
+import seaborn as sns
+import matplotlib.pyplot as plt
+
+sns.set_theme(style="ticks")
+
+sns.histplot(y)
+plt.show()
+
+###############################################################################
+# It seems better to take the log of sales rather than the sales amount itself:
+import numpy as np
+
+y = np.log(y)
+sns.histplot(y)
+plt.show()
+
+###############################################################################
+# Now, let's carry out some basic preprocessing:
+
+# Get a mask of the rows with missing values in "Publisher" and "Global_Sales"
+mask1 = X.isna()["Publisher"]
+mask2 = X.isna()["Global_Sales"]
+# And remove them
+X.dropna(subset=["Publisher", "Global_Sales"], inplace=True)
+y = y[~mask1]
+y = y[~mask2]
+
+###############################################################################
+# Extracting entity embeddings
+# ----------------------------
+# We will use the |KEN| function to extract the embeddings
+# of entities we need:
+from dirty_cat.datasets import get_ken_embeddings
+
+###############################################################################
+# KEN Embeddings are classified by types. The |KEN| function
+# allows us to specify the types to be included and/or excluded
+# so as not to load all Wikipedia entity embeddings in a table.
+#
+# In a first table, we include all embeddings with the type name "game"
+# and exclude those with type name "companies" or "developer".
+embedding_games = get_ken_embeddings(
+    types="game",
+    emb_id="39254360",
+    emb_type_id="38879442",
+    exclude="companies|developer",
+)
+
+###############################################################################
+# In a second table, we include all embeddings containing the type name
+# "game_development_companies", "game_companies" or "game_publish":
+embedding_publisher = get_ken_embeddings(
+    "game_development_companies|game_companies|game_publish",
+    emb_id="39254360",
+    emb_type_id="38879442",
+    suffix="_aux",
+)
+
+# We keep the 200 embeddings column names in a list (for the |Pipeline|):
+n_dim = 200
+
+emb_columns = [f"X{j}" for j in range(n_dim)]
+
+emb_columns2 = [f"X{j}_aux" for j in range(n_dim)]
+
+###############################################################################
+# Merging the entities
+# ....................
+#
+# We will now merge the entities from Wikipedia with their equivalent match
+# in our video game sales table:
+#
+# The entities from the 'embedding_games' table will be merged along the
+# column "Name" and the ones from 'embedding_publisher' table with the
+# column "Publisher"
+from dirty_cat import FeatureAugmenter
+
+fa1 = FeatureAugmenter(tables=[(embedding_games, "Entity")], main_key="Name")
+fa2 = FeatureAugmenter(tables=[(embedding_publisher, "Entity")], main_key="Publisher")
+
+X_full = fa1.fit_transform(X)
+X_full = fa2.fit_transform(X_full)
+
+###############################################################################
+# Prediction with regular features
```

Comment:
Minor comment: I would use "base features" rather than "regular features" for clarity. Also in the plotted figures I would remove "Variables" and update the `ylabels` to "Base features", "KEN features", and "Base + KEN features" so that it's easier to understand !